MaurerDistanceTransform/CMakeLists.txt

```
@@ -1,13 +1,11 @@
-cmake_minimum_required(VERSION 3.5)
-
 project(MaurerDistanceTransform)
 
 # Find SlicerExecutionModel
 find_package(SlicerExecutionModel REQUIRED)
 include(${SlicerExecutionModel_USE_FILE})
  
 # Find ITK
-find_package(ITK 4.4.0 REQUIRED COMPONENTS
+find_package(ITK 5.3.0 REQUIRED COMPONENTS
```

```suggestion
find_package(ITK 5.3 REQUIRED COMPONENTS
```

For sake of consistency

---

Done.